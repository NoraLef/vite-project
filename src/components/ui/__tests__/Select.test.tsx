import { userEvent, page } from '@vitest/browser/context';
import { render } from 'vitest-browser-react';
import { describe, expect, it } from "vitest";
import { Select } from '../Select';

describe("Select", () => {
    it("should render", () => {
        const { baseElement } = render(<Select />);

        expect(baseElement).toMatchSnapshot();
    });

    // it("should select", () => {
    //     render(<SelectScrollable />);

    //     const optionSelect = page.getByRole('combobox', { name: 'Select a timezone' });
    //     expect.element(optionSelect).toBeVisible();
    //     userEvent.click(optionSelect);
      
    //     const option1 = page.getByRole('option', { name: 'Eastern Standard Time (EST)' });
    //     expect.element(
    //         option1
    //     ).toBeVisible();
    //     userEvent.click(option1);
    //     expect.element(optionSelect).toHaveValue('est');
    // });
});