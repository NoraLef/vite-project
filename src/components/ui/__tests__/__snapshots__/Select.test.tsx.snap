// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`Select > should render 1`] = `
<body>
  
    
  
  


  <div>
    <select
      aria-hidden="true"
      style="position: absolute; border: 0px; width: 1px; height: 1px; padding: 0px; margin: -1px; overflow: hidden; clip: rect(0px, 0px, 0px, 0px); white-space: nowrap; overflow-wrap: normal;"
      tabindex="-1"
    >
      <option
        value=""
      />
    </select>
  </div>
</body>
`;
